keyboards/daisy/rules.mk

```diff
@@ -30,3 +30,5 @@ BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
+VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = lite
```

Comment:
This shouldnt be enabled here but in a `via` keymap, <https://caniusevia.com/docs/configuring_qmk>
```suggestion
```

Reply:
Will do